packages/runtime-common/realm.ts

```diff
@@ -116,6 +117,8 @@ export interface RealmAdapter {
 
   openFile(path: LocalPath): Promise<FileRef | undefined>;
 
+  // this should return unix time as it's the finest resolution that we can rely
+  // on across all envs
   lastModified(path: LocalPath): Promise<number | undefined>;
```

Comment:
Since we now rely on realm HTTP API for performing file reads in our indexer, that also means that we rely on the `last-modified` HTTP response header for the last modified time of the file. The HTTP spec specifies the finest resolution of this time as a second. so we need to be consistent in our `lastModified` times such that they in units of seconds and not milliseconds.